filesystem/README.md

```diff
@@ -0,0 +1,47 @@
+# Filesystem example
```

Comment:
the filesystem example is currently broken due to https://github.com/cockroachdb/cockroach/issues/3167